tests/bot/exts/filtering/test_extension_filter.py

```diff
@@ -68,6 +69,7 @@ async def test_message_with_illegal_extension(self):
 
         self.assertEqual(result, ({}, ["`.disallowed`"], {ListType.ALLOW: []}))
 
+    @pytest.mark.xfail
```

Comment:
If these tests passed before, they should pass now.

If they are no longer appropriate, please update the tests to support the new behaviour.